cluster/addons/registry/registry-svc.yaml

```diff
@@ -1,17 +1,12 @@
-apiVersion: v1
 kind: Service
+apiVersion: v1
 metadata:
-  name: kube-registry
-  namespace: kube-system
```

Comment:
why are you moving it out of kube-system?

Reply:
@roberthbailey 
I think this does not belong to the kube-system
I think he can be seen in default